File: KuantumKaos/csharp/kuantumKaos/VeriPaketi.cs
using kuantumKaos;
using System;

namespace kuantumKaos
{
    public class VeriPaketi : KuantumNesnesi
    {
        public override void AnalizEt()
        {
            Stabilite -= 5;
            Console.WriteLine("Veri içeriği okundu.");

            if (Stabilite <= 0)
            {
                throw new KuantumCokusuException(ID);
            }
        }
    }
}

File: KuantumKaos/javascript/js_kaos/VeriPaketi.js
import { KuantumNesnesi } from "./KuantumNesnesi.js";
import { KuantumCokusuException } from "./KuantumCokusuException.js";

export class VeriPaketi extends KuantumNesnesi {
    analizEt() {
        this.stabilite -= 5;
        console.log("Veri paketi analiz edildi.");

        if (this.stabilite <= 0) {
            throw new KuantumCokusuException(this.id);
        }
    }
}
